plugin/proxy/lookup.go

```diff
@@ -28,7 +28,7 @@ func NewLookupWithOption(hosts []string, opts Options) Proxy {
 		from: ".",
 		HealthCheck: healthcheck.HealthCheck{
 			FailTimeout: 5 * time.Second,
-			MaxFails:    3,
+			MaxFails:    1,
 		},
```

Comment:
Here I understand we just do what is said in the readme : max_fails is 1 by default.